modules/standard/IO.chpl

```diff
@@ -4386,35 +4385,38 @@ private inline proc markHelper(fileRW) throws {
     memory space requirements.
 
   :returns: The offset that was marked
-  :throws SystemError: if marking the fileReader failed
+  :throws SystemError: if marking the ``fileReader`` failed
  */
 proc fileReader.mark() throws do return markHelper(this);
+// In the future, we'd like to support IO transactions using context managers.
```

Comment:
Let's slap a TODO on this with a date, just to make it easier to search for and stand out more visually

Reply:
Today's date, or the date that we would expect to start working on this?

Reply:
Today's date